### DISCUSSION TITLE
global proxy

### DISCUSSION THREAD

**Discussion by mo-han** [TriliumNext/Trilium#2744] [Q&A]
i'm behind a proxy.
when copying content from a web page and paste in trilium, images fail to load.
i suppose i need to config proxy for trilium so it could grab those images from internet.
how?

**Comment by zadam** [comment]
Hi, you can set up the proxy server here:

![image](https://user-images.githubusercontent.com/617641/160239785-c66d8ce6-a56f-4066-8f79-3f3505307ca3.png)

It says it's "sync proxy" but it's also used for image download. If it's not filled then a system proxy is used.
